esi_leap/manager/service.py

```diff
@@ -70,13 +70,14 @@ def _fulfill_leases(self):
                     lease.fulfill(self._context)
                 except Exception as e:
                     LOG.info("Error fulfilling lease: %s", e)
```

Comment:
Also probably out of scope for this PR but would we want to include the exception type in the error log? If we have an exception with a name like OutOfMemoryException with a faultstring that's blank or unhelpful without the class name, we'd be leaving out key context.

Reply:
Ah, you're right - I'm going to open up a new issue for this one for someone else to fix!